README.md

```diff
@@ -90,18 +90,20 @@ exports.hello = (event, context, callback) => {
 };
 ```
 
-You can just create a file with following informations:
+In order to deploy this function, we need the credentials with permissions to access Aliyun Function Compute. 
+Please create a `credentials` file and configure the credentials in it. 
+Here is an example `credentials` file:
 
 ```ini
 [default]
-aliyun_access_key_id = nA5hjMhbg9BOoVo
-aliyun_access_key_secret = 098f6bcd4621d373cade4e832627b4f6
-aliyun_account_id = 1504163990726
+aliyun_access_key_id = xxxxxxxx
+aliyun_access_key_secret = xxxxxxxxxxxxxxxxxxxx
+aliyun_account_id = 1234567890
 ```
 
-The `aliyun_access_key_secret` and `aliyun_access_key_id` you can found at https://ak-console.aliyun.com/?#/accesskey . If no any Access Key, you can create a Access Key or use subaccount Access Key. And `aliyun_account_id` can be found at https://account-intl.console.aliyun.com/?#/secure .
-
-After create the credentials file, please make sure pointing the value of the `credentials` field in `serverless.yml` to it.
+Find the `aliyun_access_key_secret` and `aliyun_access_key_id` from https://ak-console.aliyun.com/?#/accesskey, or create an Access Key or use sub-account Access Key. 
```

Comment:
You can find the `aliyun_access_key_secret` and `aliyun_access_key_id` from https://ak-console.aliyun.com/?#/accesskey. You can also chose to create an Access Key or use sub-account Access Key.